_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -471,7 +471,7 @@ ggplot() +
 > > colors <- c("purple", "springgreen", "yellow", "brown", "navy")
 > > ```
 > > Now we can create our plot: 
-> > ```{r colored-state-boundaries}
+> > ```{r colored-state-boundaries fig.cap="Map of the continental United States where the state lines are colored by region."}
```

Comment:
```suggestion
> > ```{r colored-state-boundaries, fig.cap="Map of the continental United States where the state lines are colored by region."}
```